modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/xmlformatter/format_xml.es.js

```diff
@@ -0,0 +1,150 @@
+import {isString} from 'metal';
```

Comment:
Move util to root `util` dir, as this is the only function there is no need to have `xmlformatter` dir. Update path in test.